dc_signup_form/urls.py

```diff
@@ -1,5 +1,9 @@
 from django.conf.urls import url
 
+from .constants import (
+    MAILING_LIST_FORM_PREFIX,
+    ELECTION_REMINDERS_FORM_PREFIX
+)
```

Comment:
Can you also update the documentation on custom routes: https://github.com/DemocracyClub/dc_signup_form#usage If we now need to pass a prefix when we are creating a route, the docs should reflect that (although as noted elsewhere, if its possible to make a wrapper or abstraction, maybe this is unnecessary). Either way, the docs should reflect the changes in this PR.